backend/src/model/user.js

```diff
@@ -1,21 +1,54 @@
 const uuid = require("uuid");
 const crypto = require("crypto");
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../Database/index");
 
 class User {
-
   constructor(id) {
+    this.User;
     if (id) {
       this.id = id;
     } else {
       this.id = uuid.v4();
     }
   }
 
+  create(user) {
```

Comment:
acho que seria legal separar aqui em mais arquivos (tipo repository e database)

Reply:
To precisando desse PR, vou mergear ele e aí corrige esses pontos no próximo 😛